app/src/lib/stats/stats-store.ts

```diff
@@ -706,22 +715,22 @@ export class StatsStore implements IStatsStore {
   }
 
   /** Record that user dismissed diverging branch notification */
-  public async recordDivergingBranchBannerDismissal(): Promise<void> {
+  public recordDivergingBranchBannerDismissal(): Promise<void> {
```

Comment:
i'd prefer we didn't make all of these signature changes in this PR, but i'm not going to consider it blocking

Reply:
Looks like `changesTakenToNewBranchCount` is now failing. Was working as of 481ab726d6c2c1f50daf0051cba5d707beb3d4c3

Reply:
ok reverted 4743c6a with b9e42bc

Reply:
> i'd prefer we didn't make all of these signature changes in this PR, but i'm not going to consider it blocking

Yea, that was my thinking in https://github.com/desktop/desktop/pull/7257#discussion_r288309274.

Reply:
> Looks like `changesTakenToNewBranchCount` is now failing. Was working as of [481ab72](https://github.com/desktop/desktop/commit/481ab726d6c2c1f50daf0051cba5d707beb3d4c3)

I'd like to confirm that b9e42bc05b3fa681c6d49b9a1078a312a0f9f76a is the right fix here, and is separate to the discussion about the `async` modifier here. @tierninho are you good with your QA work in here?

Reply:
Sorry, missed this comment. Yes, all is good on my end.